params/hooks_libevm.go

```diff
@@ -100,6 +100,7 @@ func makePrecompile(contract contract.StatefulPrecompiledContract) libevm.Precom
 			panic(err) // Should never happen
 		}
 		var predicateResults predicate.BlockResults
+		rules := GetRulesExtra(env.Rules()).AvalancheRules
```

Comment:
Nit: Typically it'd be cleanest to put the var definition as close as possible to its use below. In this case, it's a bit of a gray area because putting it here aligns it most closely with coreth (which needs it for `PredicateBytesFromExtra`).

Nonetheless, I generally think that best practices shouldn't be bypassed for contextual reasons not apparent in this code repo itself, so would lean towards putting it as close as possible to its use for now.